worker/src/RTC/DtlsTransport.cpp

```diff
@@ -1022,9 +1047,24 @@ namespace RTC
 			MS_WARN_TAG(
 			  dtls, "OpenSSL SSL_write() wrote less (%d bytes) than given data (%zu bytes)", written, len);
 		}
+	}
+
+	void DtlsTransport::SendDtlsData(const uint8_t* data, size_t len)
+	{
+		MS_TRACE();
+
+		MS_DEBUG_DEV("%zu bytes of DTLS data ready to be sent", len);
 
-		// Send data.
-		SendPendingOutgoingDtlsData();
+		// Notify the listener.
+		this->listener->OnDtlsTransportSendData(this, data, len);
+
+		// Clear the BIO buffer.
+		// NOTE: Here we now that OpenSSL called the onSslBioOut() callback with
```

Comment:
```suggestion
		// NOTE: Here we now that OpenSSL called the onSslBioOut() callback
```

Reply:
No.

> `BIO_set_callback_ex()` and `BIO_get_callback_ex()` set and retrieve the BIO callback. The callback is called during most high-level BIO operations. It can be used for debugging purposes to trace operations on a BIO or **to modify its operation**.

Reply:
Fixed in local branch.